sql/common/src/test/scala/org/apache/sedona/sql/rasteralgebraTest.scala

```diff
@@ -470,6 +470,12 @@ class rasteralgebraTest extends TestBaseScala with BeforeAndAfter with GivenWhen
       assert(writtenBinary1.length > writtenBinary2.length)
     }
 
+    it("Passed RS_Base64") {
```

Comment:
This is not a correct output of Base64. I believe it should start with "data:PNG". Please double check.